modes/longitudinal/src/initToolGroups.js

```diff
@@ -243,6 +243,9 @@ function initMPRToolGroup(extensionManager, toolGroupService, commandsManager, m
       { toolName: toolNames.Angle },
       { toolName: toolNames.CobbAngle },
       { toolName: toolNames.PlanarFreehandROI },
+      { toolName: toolNames.SplineROI },
+      { toolName: toolNames.LivewireContour },
+      { toolName: toolNames.CalibrationLine },
```

Comment:
i don't think calibration is used (works) in volume, maybe take this out. Also see if any other mode need to be edited by adding spline and livewire